index.html

```diff
@@ -1,6 +1,57 @@
-<head>
-  <script type="text/javascript" src="https://firefoxux.github.io/SharedBanner/fxux-banner.min.js"></script>
```

Comment:
Maybe?  For code-sharing, I suggest we stick with this one for now, and file a followup bug to figure out how to have a static one later…